docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -101,129 +101,221 @@ In the case of errors, include human-readable error messages in the response pay
+#### Types of configuration options
+These are the types of configuration options that we allow:
+- Text
+- Checkbox 
+- Select
+- Multi-select
+- Password
 
-`https://example.com/check_connection_options`
+Specify the following for each configuration option when you [sign up] (#sign-up): 
+- **Label**: Text that merchants see when they connect in the control panel
+- **Code**: Unique code for the option. Use snake case.
+- **Required**: Whether the option is required 
+- **Type**: Type of option
 
-```http lineNumbers
+For select and multi-select types, specify values that you want available.
 
-X-Auth-Token: {{ACCESS_TOKEN}}
-Content-Type: application/json
-Accept: application/json
-{
-  "connection_options": {
-    "account_id": "a1ty"
-  }
-}
+Below are examples of what you can specify for various configuration types:
+
+<!-- 
+type: tab
+title: Text 
+-->
+
+```text title="Example connection option" lineNumbers 
+- Label: Display Name
+- Required: true
+- Type: text
 ```
 
-**Example response**
+<!-- 
+type: tab
+title: Checkbox 
+-->
 
-```json
-{
-  "valid": false,
-  "messages": [
-    {
-      "text": "Your account ID is invalid",
-      "type": "ERROR"
-    }
-  ]
-}
+```text title="Example connection option" lineNumbers 
+- Label: Shipping Rate
+- Required: false
+- Type: Checkbox
+- Values: Enable Expedited Shipping
 ```
 
-<!-- theme: info -->
-> #### Credential validation
-> It is best practice to authenticate the user and store against your database or the downstream provider service.
+<!-- 
+type: tab
+title: Select 
+-->
 
-Once you install the app, it will be made available for configuration by merchants and API users. A merchant can navigate to the Shipping Manager and enable, configure, and disable the carrier for any defined zone.
+```text title="Example connection option" lineNumbers 
+- Label: Packaging Type
+- Required: false
+- Type: select
+- Values: Anti-corrosive, Plastics, Cardboard
+```
+
+<!-- 
+type: tab
+title: Multi-select
+-->
+
+```text title="Example connection option" lineNumbers 
+- Label: Packaging Method
+- Required: false
+- Type: Multi-select
+- Values: Extra primary packaging, Extra secondary packaging, Extra tertiary packaging
+```
+
+<!-- 
+type: tab
+title: Password 
+--> 
+
+```text title="Example connection option" lineNumbers 
+- Label: Password
+- Required: true
+- Type: password
+```
+
+<!-- type: tab-end -->
+
+
+## Sign up
+After you develop the app, sign up to be a shipping provider. Submit your app to BigCommerce and BigCommerce will register your app as a shipping provider. 
+
+### What you should provide BigCommerce
+Send an email to [ShippingProviderAPI@bigcommerce.com](mailto:shippingproviderapi@bigcommerce.com) that includes the following information:
+
+- Name of app
+- [Your app ID](#your-app-id)
+- Your email
+- A description of the app. The description is displayed to merchants in the control panel.
+- Logo: A 70x70 pixel logo that represents the shipping carrier app

```

Comment:
The logo property can also be optional as it is not required for those who want to keep their app in a draft state